rustls/src/msgs/persist.rs

```diff
@@ -188,10 +186,14 @@ impl Tls12ClientSessionValue {
         }
     }
 
-    pub(crate) fn ticket(&mut self) -> Arc<PayloadU16> {
+    pub(crate) fn ticket(&self) -> Arc<PayloadU16> {
         Arc::clone(&self.common.ticket)
     }
 
+    pub(crate) fn ticket_bytes(&self) -> &[u8] {
+        self.common.ticket.0.as_slice()
+    }
```

Comment:
If the return value should be treated specially if it `is_empty()`, suggest yielding an `Option` here.